packages/entityanalytics_ad/data_stream/user/elasticsearch/ingest_pipeline/entity.yml

```
@@ -273,6 +273,59 @@ processors:
       field: user.id
       copy_from: activedirectory.user.object_sid
 
+  - set:
+      field: activedirectory.user.logon_script_enabled
+      value: true
+      if: "ctx.activedirectory?.user?.uac_list?.contains('SCRIPT')"
```

```suggestion
      if: ctx.activedirectory?.user?.uac_list?.contains('SCRIPT') == true
```
It is possible for the expression as it stands to resolve to `null` which will cause the pipeline to error out. Same below.

---

Changed the condition on them all